nio/client/async_client.py

```diff
@@ -3441,6 +3441,35 @@ async def room_upgrade(self,
 
         return RoomUpgradeResponse(new_room.room_id)
 
+    @logged_in
+    async def update_room_topic(
+        self,
+        room_id: str,
+        topic: str,
+        state_key: str = "",
```

Comment:
I don't think anyone uses any `state_key` for the topic, if someone really needs this they can send a state event using the generic method.

Let's remove this.

Reply:
Fixed in e0fdf67.